doc/shelley-genesis.md

```diff
@@ -955,7 +955,7 @@ be 30 seconds into the future, and then we can start our nodes. This assumes
 you did not do any manual tweaking of the generated `genesis.json`, as it will
 be overwritten.
 ```
-$ cardano-cli shelley genesis create --genesis-dir example/
+$ cardano-cli shelley genesis create --genesis-dir example/ --supply 1000000000
```

Comment:
This example does not set an initial supply, but one just a little later does.

Rather than adding a supply amount here, I think it would be better to add a note and explaining that it will be set in a later coommand.

I have a patch in a WIP branch that adds the extra comment.

Reply:
The confusion came for me from de manually / semi-automagically parts : I started with the manually and skipped automagically, so I didn't type the later automagical command. le mixed manual / automagical is a bit confusing. Maybe giving the later command in both format would be better.